engines/e_devcrypto.c

```diff
@@ -41,7 +41,9 @@ static int cfd = -1;
 #define DEVCRYPTO_USE_SOFTWARE        1 /* allow software drivers */
 #define DEVCRYPTO_REJECT_SOFTWARE     2 /* only disallow confirmed software drivers */
 
+#ifndef DEVCRYPTO_DEFAULT_USE_SOFTDRIVERS
 #define DEVCRYPTO_DEFAULT_USE_SOFTDRIVERS DEVCRYPTO_REJECT_SOFTWARE
```

Comment:
Coding style: Because of the guard, this needs to be indented by inserting a space between `#` and `define`